src/pages/pt-br/guides/fonts.md

```diff
@@ -0,0 +1,118 @@
+    }
+    ```
+
+    :::note
+    Não incluímos 'public' na URL de origem das fontes porque todos os arquivos em 'public' são adicionados ao diretório raiz do seu site.
+    :::
+
+3. Utilize a declaração `font-family` da declaração `@font-face` para estilizar elementos em seu componente ou layout. Nesse exemplo, o título `<h1>` terá a fonte personalizada aplicada, enquanto que o parágrafo `<p>` não terá.
+
+    ```astro {10-12}
+    ---
+    // src/pages/example.astro
+    ---
+
+    <h1>Em uma galáxia distante, muito distante...</h1>
+
+    <p>Fontes personalizadas tornam meus cabeçalhos muito mais legais!</p>
+
+    <style>
+    h1 {
+      font-family: 'DistantGalaxy', sans-serif;
+    }
+    </style>
+    ```
+
+## Usando Fontsource
+
+O projeto [Fontsource](https://fontsource.org/) simplifica o uso das fontes do Google e outras fontes de código aberto. Ele fornece módulos npm que você pode instalar para as fontes que você deseja utilizar.
+
+1. Encontre a fonte que você quer usar no [Catálogo de fontes](https://fontsource.org/fonts). Para este exemplo, vamos usar a fonte [Twinkle Star](https://fontsource.org/fonts/twinkle-star).
+
+2. Instale o pacote para a sua fonte escolhida.
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm i @fontsource/twinkle-star
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm i @fontsource/twinkle-star
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn add @fontsource/twinkle-star
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+    :::tip
+    Você encontrará o nome correto do pacote na seção "Instalação rápida (Quick Installation)" de cada página de fontes no site da Fontsource. Ele começará com `@fontsource/` seguido do nome da fonte.
+    :::
+
+3. Importe o pacote de fontes no layout ou componente onde você deseja usar a fonte. Geralmente, você vai querer fazer isso em um componente de layout comum para garantir que a fonte esteja disponível em todo o seu site.
+
+    A importação do pacote acrescentará automaticamente as regras `@font-face` necessárias para a configuração da fonte.
+
+    ```astro
+    ---
+    // src/layouts/BaseLayout.astro
+    import '@fontsource/twinkle-star';
+    ---
+    ```
+
+4. Use a `font-family` como mostrado na página de fontes do Fontsource. Isto funcionará em qualquer lugar onde você possa escrever CSS em seu projeto Astro.
+
+    ```css
+    h1 {
+      font-family: "Twinkle Star", cursive;
+    }
+    ```
+
+## Materiais extras
+
+### Adicione fontes com Tailwind
+
+Se você estiver usando a [integração Tailwind](/en/guides/integrations-guide/tailwind/), você pode adicionar uma declaração `@font-face` para uma fonte local ou usar usar a estratégia de `import` da Fontsource para registrar sua fonte. Então, siga [os documentos do Tailwind sobre como adicionar famílias de fontes personalizadas](https://tailwindcss.com/docs/font-family#using-custom-values).
```

Comment:
Same here that we should link to the `/pt-br/` guide (even if it’s not translated yet) to keep readers on their own language’s version of docs.

```suggestion
Se você estiver usando a [integração Tailwind](/pt-br/guides/integrations-guide/tailwind/), você pode adicionar uma declaração `@font-face` para uma fonte local ou usar usar a estratégia de `import` da Fontsource para registrar sua fonte. Então, siga [os documentos do Tailwind sobre como adicionar famílias de fontes personalizadas](https://tailwindcss.com/docs/font-family#using-custom-values).
```